src/libs/actions/Report.js

```diff
@@ -115,6 +115,17 @@ function getUnreadActionCount(report) {
     return Math.max(0, unreadActionCount);
 }
 
+/**
+ * Used before merging Report data.
+ * Cut last message to specific length, because we don't need full last message
+ *
+ * @param {String} lastMessage
+ * @returns {String}
+ */
+function cutLastMessage(lastMessage) {
+    return lastMessage.substr(0, CONST.REPORT.MAX_LAST_MESSAGE_LENGTH);
+}
+
```

Comment:
Let's move this to reportUits  as `formatReportLastMessageText`

Reply:
Done